src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -230,163 +255,135 @@ const AnalysisCard = ({
   ])
 }
 
-const Analyses = _.flow(
-  forwardRefWithName('Analyses'),
-  requesterPaysWrapper({
-    onDismiss: () => Nav.history.goBack()
-  }),
-  wrapWorkspace({
-    breadcrumbs: props => breadcrumbs.commonPaths.workspaceDashboard(props),
-    title: 'Analyses',
-    activeTab: 'analyses'
-  }),
-  withViewToggle('analysesTab')
-)(({
-  name: workspaceName, namespace, workspace, workspace: { accessLevel, canShare, workspace: { cloudPlatform, workspaceId, googleProject, bucketName } },
+const activeFileTransferMessage: React.ReactElement<'div'> = div({
```

Comment:
Another case where the type can be inferred and doesn't need to be declared.

Reply:
Its inferred as `React.ReactElement<any, any>` otherwise, and it is a bit helpful to know its a div where its used